pkgs/by-name/ae/aegisub/package.nix

```
@@ -1,158 +1,295 @@
+    (lib.mesonBool "local_boost" true)
+
+    (lib.mesonOption "default_audio_output" "auto")
+    (lib.mesonEnable "alsa" alsaSupport)
+    (lib.mesonEnable "openal" openalSupport)
+    (lib.mesonEnable "libpulse" pulseaudioSupport)
+    (lib.mesonEnable "portaudio" portaudioSupport)
+    (lib.mesonEnable "directsound" false)
+    (lib.mesonEnable "xaudio2" false)
 
-  preConfigure = ''
-    export FORCE_GIT_VERSION=${finalAttrs.version}
-  '';
+    (lib.mesonEnable "ffms2" true)
+    (lib.mesonEnable "avisynth" true)
+    (lib.mesonEnable "bestsource" true)
+    (lib.mesonEnable "vapoursynth" true)
+
+    (lib.mesonEnable "fftw3" true)
+    (lib.mesonEnable "uchardet" true)
+    (lib.mesonEnable "csri" true)
+    (lib.mesonEnable "hunspell" spellcheckSupport)
+  ];
 
-  cmakeBuildDir = "build-directory";
+  preConfigure =
+    ''
+      cp -r --no-preserve=mode ${bestsource} subprojects/bestsource
+      cp -r --no-preserve=mode ${AviSynthPlus} subprojects/avisynth
+      cp -r --no-preserve=mode ${vapoursynth} subprojects/vapoursynth
+      cp -r --no-preserve=mode ${ffms2} subprojects/ffms2
+      sed -i '28i\#include <string>' subprojects/bestsource/src/videosource.h
 
-  strictDeps = true;
+      mkdir subprojects/packagecache
+      cp -r --no-preserve=mode ${gtest} subprojects/packagecache/gtest-1.8.1.zip
+      cp -r --no-preserve=mode ${gtest_patch} subprojects/packagecache/gtest-1.8.1-1-wrap.zip
+      cp -r --no-preserve=mode ${boost} subprojects/packagecache/boost_1_74_0.tar.gz
+
+      meson subprojects packagefiles --apply bestsource
+      meson subprojects packagefiles --apply avisynth
+      meson subprojects packagefiles --apply vapoursynth
+      meson subprojects packagefiles --apply ffms2
+    ''
+    + lib.optionalString stdenv.isDarwin ''
+      cp -r --no-preserve=mode ${wxWidgets} subprojects/wxWidgets
+      meson subprojects packagefiles --apply wxWidgets
+      substituteInPlace subprojects/wxWidgets/configure --replace \
+        'SEARCH_INCLUDE=' 'DUMMY_SEARCH_INCLUDE='
+      substituteInPlace subprojects/wxWidgets/configure --replace \
+        'SEARCH_LIB=' 'DUMMY_SEARCH_LIB='
+      substituteInPlace subprojects/wxWidgets/configure --replace \
+        /usr /no-such-path
+      substituteInPlace subprojects/wxWidgets/configure --replace \
+        'ac_cv_prog_SETFILE="/Developer/Tools/SetFile"' \
+        'ac_cv_prog_SETFILE="${setfile}/bin/SetFile"'
+      substituteInPlace subprojects/wxWidgets/configure --replace \
+        "-framework System" "-lSystem"
+      substituteInPlace subprojects/wxWidgets/build/cmake/files.cmake --replace \
+        "wx/thrimpl.cpp" ""
+      meson subprojects packagefiles --apply wxWidgets
+    '';
 
   meta = {
-    homepage = "https://github.com/wangqr/Aegisub";
-    description = "Advanced subtitle editor; wangqr's fork";
+    homepage = "https://github.com/arch1t3cht/Aegisub";
+    description = "An advanced subtitle editor; arch1t3cht's fork";
```

```suggestion
    description = "Advanced subtitle editor; arch1t3cht's fork";
```

Recent directives do not allow indefinite articles at the beginning.